creepy/subprocess/pypen.py

```diff
@@ -141,6 +146,23 @@ def detach(self):
         except (OSError, FileNotFoundError, AttributeError):
             pass
 
+    @lru_cache(maxsize=1)
+    def _make_proxy_type(self):
```

Comment:
This should be a static function that accepts `interface` param and returns type with constructor which accepts Pypen or pypen.request.